.github/workflows/examples.yml

```diff
@@ -0,0 +1,50 @@
+name: Examples
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.6'
+          - '1'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
```

Comment:
Since, these are documentation, is it worth testing all possible configuration?
Besides, the cutest benchmark will be trouble on Windows (and Mac https://github.com/JuliaSmoothOptimizers/CUTEst.jl/pull/327 ?)

Reply:
Yes, I would like to test all combinations so we can detect if/when they break.